src/DetailsView/components/failure-instance-panel-control.tsx

```diff
@@ -115,10 +130,73 @@ export class FailureInstancePanelControl extends React.Component<FailureInstance
         );
     }
 
+    protected getSelectorAndSnippet = (): JSX.Element => {
```

Comment:
any particular reason this function is `protected`??? are we extending this class somewhere?

Reply:
great point, switching to private since it's internal use only